tests/testthat/setup.R

```diff
@@ -105,7 +105,8 @@ if (Sys.getenv("CDM5_ORACLE_SERVER") != "") {
       user = Sys.getenv("CDM5_ORACLE_USER"),
       password = URLdecode(Sys.getenv("CDM5_ORACLE_PASSWORD"))
     ),
-    cdmDatabaseSchema = Sys.getenv("CDM5_ORACLE_CDM54_SCHEMA"),
+    # cdmDatabaseSchema = Sys.getenv("CDM5_ORACLE_CDM54_SCHEMA"),
+    cdmDatabaseSchema = "CDMV54",
```

Comment:
Should we fix the secret to be uppercase?

Reply:
Yes please. I think you'll need to do it as I don't have admin access.